code/renderers/vue3/src/docs/sourceDecorator.ts

```diff
@@ -120,13 +119,10 @@ export const shouldSkipSourceCodeGeneration = (context: StoryContext): boolean =
   }
 
   const isArgsStory = context?.parameters.__isArgsStory;
-  const isDocsViewMode = context?.viewMode === 'docs';
 
   // never render if the user is forcing the block to render code, or
   // if the user provides code, or if it's not an args story.
-  return (
-    !isDocsViewMode || !isArgsStory || sourceParams?.code || sourceParams?.type === SourceType.CODE
```

Comment:
I think showing this on story view. mode should be opt-in, so you add new field the docs parameter

Reply:
For all other renderers it is always generated, we currently only check for the view mode for Vue.
Also in my opinion, the Code panel is a big benefit for users. What do you think about adding an opt-out preview parameter so the tab can be disabled?

Reply:
I agree that the Code panel is a good addition and will be beneficial for users. Since Docs is an opt-in feature, I'll leave it to @shilman to decide on that part. I'm okay with everything else in the PR.